data/korath/korath ships.txt

```diff
@@ -35,20 +35,38 @@ ship "Korath Raider"
 		"Korath Grab-Strike" 4
 		"Korath Banisher"
 		"Korath Warder"
-		
+
```

Comment:
Don't change the whitespace.

Reply:
I didn't mean too computer did it automatically, fixing now

Reply:
I think that I fixed the errors.